packages/ui/components/form/inputs/Input.tsx

```diff
@@ -296,25 +296,7 @@ const PlainForm = <T extends FieldValues>(props: FormProps<T>, ref: Ref<HTMLForm
             });
         }}
         {...passThrough}>
-        {
-          /* @see https://react-hook-form.com/advanced-usage/#SmartFormComponent */
-          React.Children.map(props.children, (child) => {
```

Comment:
This was actually breaking all our forms 🤯  
I've been searching everywhere why my new forms in `Console` where NOT working (state got lost of the form fields), but all the fields in web actually WERE working. 

Well.... It turns out that the fields in `Web` also don't work if you put them as a direct child of the form. In `Web` we seem to have gotten used to wrap all our inputs at least in a `div` or `fieldset` to get it to work. So instead of also wrapping my fields in `Console` in `divs` as well, I opted to remove it. 

Would love your opinions on this to know fur sure I didn't went crazy 😁

Reply:
Looking at https://codesandbox.io/s/react-hook-form-smart-form-component-forked-iq89z?file=/src/components.js:109-117 there was a mistake made in the manner we handle non-array situations. 

Adopting the following logic seems ideal

```javascript
{Array.isArray(children)
    ? children.map((child) => {
        return child.props.name
          ? React.createElement(child.type, {
              ...{
                ...child.props,
                register,
                key: child.props.name
              }
            })
          : child;
      })
    : children}
```

Reply:
I tried that as well. Didn't work either 🤷‍♂️

Also replaced createElement with cloneElement, since that also keeps references for the children. That actually DID work. But I find that quite a risky change to add randomly in this PR. That's why I opted to actually delete it since it is broken now and we don't use it.

Reply:
Yeah I think this abstraction isn't clear enough and we should get rid of it meanwhile we come up with a better one.